locales/it/pokemon.json

```diff
@@ -333,32 +333,32 @@
   "Magikarp Calico (Orange, White, Black)": "",
   "Magikarp Calico (White, Orange)": "",
   "Magikarp Calico (Orange, Gold)": "",
```

Comment:
The two color Calicos seem to be `(first e second)` on that wiki link, instead of comma separate like we would get if left alone.

Reply:
Good catch lol